.github/workflows/tests.yml

```
@@ -21,6 +21,8 @@ jobs:
             postgres: 12
           - python: 3.9
             postgres: 13
+          - python: 3.10.0-beta.4
```

At least for GHA, this is simpler and will pick up the latest, for example it's now RC1:

```suggestion
          - python: 3.10-dev
```